app/helpers/organisation_helper.rb

```diff
@@ -7,7 +7,7 @@ def show_organisation_link?
       current_user.organisation.present?
   end
 
-  def hide_draft_records?
+  def lead_school_user?
```

Comment:
Should this be somewhere else? I generally try and avoid helpers as it always seemed a bit 'big utility class' anti pattern to me but we do use them elsewhere. Doesn't seem appropriate to be on the organisation helper though. I realise they just get included but this isn't the place I'd go to find it. `UserHelper` maybe?